frontend/src/pages/_document.tsx

```diff
@@ -1,53 +1,8 @@
-import RawDocument, {
-  DocumentContext,
-  DocumentInitialProps,
-  Head,
-  Html,
-  Main,
-  NextScript,
-} from "next/document";
-import { ServerStyleSheet } from "styled-components";
+import RawDocument, { Head, Html, Main, NextScript } from "next/document";
 
 const OG_PAGE_TITLE = "Cellxgene Data Portal";
 
 export default class Document extends RawDocument {
-  static async getInitialProps(
```

Comment:
Just curious, why is this function going away?

Reply:
Ah sorry for missing to document this change!

`styled-components` needed the extra config to server side render the stylesheets, but `@emotion` handles it automatically, so less code for us to maintain!